JSDemos/Demos/Drawer/LeftOrRightPosition/description.md

```diff
@@ -1,5 +1,18 @@
-The <b>Drawer</b> is&nbsp;a&nbsp;dismissible or&nbsp;permanently visible panel used for navigation in&nbsp;responsive web applications. On&nbsp;opening, the <b>Drawer</b> can interact with the view in&nbsp;several modes: <i>"shrink"</i>, <i>"push"</i>, and <i>"overlap"</i>. The view's content (an&nbsp;HTML string in&nbsp;this demo) is&nbsp;part of&nbsp;the <b>Drawer</b>'s markup. The <b>Drawer</b>'s content is&nbsp;defined using a&nbsp;<a href="/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#template">template</a>. Note that the <b>Drawer</b> does not interact with the toolbar because the latter is&nbsp;not defined in&nbsp;the <b>Drawer</b>'s markup.
- 
-In&nbsp;this demo, you can use radio buttons under the <b>Drawer</b> to&nbsp;change its position and behavior.
+Drawer is a dismissible or permanently visible panel that users can use for navigation in applications. 
 
-To get started with the DevExtreme Drawer component, refer to the following tutorial for step-by-step instructions: [Getting Started with Navigation Drawer](/Documentation/Guide/UI_Components/Drawer/Getting_Started_with_Navigation_Drawer/).
\ No newline at end of file
+In this demo, when users click on a toolbar item, it opens the Drawer component. The component can interact with the view in the following modes (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 
```

Comment:
A few reasons why I wanted to edit: 

- The first sentence refers to "users" - some third party... But we can (and should) speak directly to the reader in demo description. They are the developers who came here to try these demos. So we can instruct them what to do. 
- The first sentence contains a couple of comma-separated clauses. 
- "it" is a bit vague. Who opens the drawer? The "item" opens? Or the action of the "click" opens?
- "a toolbar item" sounds like "any toolbar item". Before I looked at the demo, I assumed I would find several buttons and I can click any of them. Definite article ("the") would be the proper choice in this case. I can even be specific and say "the 'Hamburger' button". (And it's not that important if it's part of a toolbar.)
- "interact with the view" - is too generic. Once again, I wanted to be a bit more specific and mention that actions affect the layout in different ways.

```suggestion
This demo displays the 'Hamburger' button that opens or closes the Drawer component. Each Drawer visibility change can adjust the layout in the following ways (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 
```

or 

```suggestion
To open or close the Drawer component in this demo, click the 'Hamburger' button. Each Drawer visibility change can adjust the layout in the following ways (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 
```